details.html

```diff
@@ -0,0 +1,26 @@
+<%inherit file="base_app.htm" />
+
+<%block name="middle">
+<div id="section">
+    ## %for p in products:
```

Comment:
Is this for loop commented out? You could probably delete it if you aren't using it to make it easier to understand your logic

Reply:
Oh yes, I decided to take it out but totally forgot to remove it! Thanks!